lib/helpers/index.js

```diff
@@ -0,0 +1,162 @@
+const fs = require('fs');
+const path = require('path');
+const actionsNames = fs
+    .readdirSync(path.join(__dirname, 'actions'))
+    .map(fileName => fileName.replace(/\.js$/, ''));
+const transformsNames = fs
+    .readdirSync(path.join(__dirname, 'transforms'))
+    .map(fileName => fileName.replace(/\.js$/, ''));
+const rulesNames = fs
+    .readdirSync(path.join(__dirname, 'rules'))
+    .map(fileName => fileName.replace(/\.js$/, ''));
+
+const helpers = {
+    serialize(langoose) {
+        return JSON.parse(JSON.stringify(langoose));
```

Comment:
we always have to serialize rules written on langoose, before passing them to goose, to remove defined properties (modifiers)